passkey-api/introduction.mdx

```diff
@@ -0,0 +1,229 @@
+    	const creationOptions = await fetch("/passkey/start-registration", {
+    		method: "POST",
+    	}).then((res) => res.json());
+
+    	// Open "create passkey" dialog
+    	const credential = await create(creationOptions);
+
+    	// User successfully created a passkey on their device.
+    	//
+    	// The resulting `credential` object needs to be sent back to the
+    	// Passkey API as-is, through our backend:
+    	//
+    	// frontend → backend → passkey API
+    	return fetch("/passkey/finalize-registration", {
+    		method: "POST",
+    		headers: { "Content-Type": "application/json" },
+    		body: JSON.stringify(credential),
+    	});
+    }
+    ```
+
+    Here's what the whole flow looks like:
+
+    <Frame>
+    	<img src="/images/passkey-api/register-flow.png" />
+    </Frame>
+
+    As you can see, there's two steps here ("start" and "finalize"), which pass through the frontend, backend, and Passkey API.
+
+    The process looks very, very similar for logging in:
+
+    &nbsp;
+
+3. **Add endpoints to start and finish logging in**
+
+    Similar to how registering a passkey is a two-step process, so is logging in.
+
+    **Backend:**
+
+    ```jsx
+    app.post("/passkey/start-login", async (req, res) => {
+    	const loginOptions = await fetch(baseUrl + "/login/initialize", {
+    		method: "POST",
+    		headers,
+    	}).then((res) => res.json());
+
+    	// loginOptions is an object that can directly be passed to get()
+    	// (the function that opens the "select passkey" dialog)
+    	// in the frontend.
+    	res.json(loginOptions);
+    });
+
+    app.post("/passkey/finalize-login", async (req, res) => {
+    	const data = await fetch(baseUrl + "/login/finalize", {
+    		method: "POST",
+    		headers,
+    		body: JSON.stringify(req.body), // Credential the user selected
+    	}).then((res) => res.json());
+
+    	// Like when registering, data.token is a JWT that contains claims
+    	// about the user (see above)
+
+    	res.redirect("/success");
+    });
+    ```
+
+    **Frontend:**
+
+    ```jsx
+    async function loginWithPasskey() {
+    	const loginOptions = await fetch("/passkey/start-login", {
+    		method: "POST",
+    	}).then((res) => res.json());
+
+    	// Open "select passkey" dialog
+    	const credential = await get(loginOptions);
+
+    	// User selected a passkey to use.
+    	//
+    	// The returned `credential` object needs to be sent back to the
+    	// Passkey API as-is.
+    	return fetch("/passkey/finalize-login", {
+    		method: "POST",
+    		headers: { "Content-Type": "application/json" },
+    		body: JSON.stringify(credential),
+    	});
+    }
+    ```
+
+    Note that for logging in, the server can also talk to the Passkey API directly, instead of going through your backend first. Whether you go with the server-first or client-first approach is up to preference. See [Client-First Login Flow](client-first-login-flow)
```

Comment:
same the link can be modified to, `[Client-First Login Flow](/passkey-api/client-first-login-flow)`

Reply:
which one do you prefer? with `/passkey-api/` or relative?